### PR TITLE
Heuristics docstrings update for read the docs.

### LINKED ISSUE #330: docs: revamp to new google style strings for RTD: heuristic nodes

### PR DESCRIPTION
**Pull request by leeping-ng**
Closes #330

### CODE REVIEW

File: peekingduck/pipeline/nodes/heuristic/check_nearby_objs.py [hunk 1]
```diff
@@ -24,7 +24,27 @@
 
 
 class Node(AbstractNode):
-    """Node that checks if any objects are near to each other"""
+    """Node that checks if any objects are near each other.
+
+    It does so by comparing the 3D location of all objects to see which ones are near
+    each other. If one object is near another, both would be tagged.
```
Comment (sidney-tio) [inline peekingduck/pipeline/nodes/heuristic/check_nearby_objs.py:30]:
```suggestion
    each other. If the distance between two objects exceeds the minimum threshold, both would be tagged as near.
```

File: docs/_templates/class.rst [hunk 2]
```diff
@@ -39,6 +39,9 @@
 .. |obj_groups| replace:: ``obj_groups`` (:obj:`list`): a list of integers, representing the assigned
    group number of an object associated with a detected bounding box.
 
+.. |large_groups| replace:: ``large_groups`` (:obj:`list`): a list of integers, representing the group numbers
```
[the remark this answers is not in the document]
Comment (leeping-ng) [reply docs/_templates/class.rst]: Updated

File: peekingduck/pipeline/nodes/heuristic/bbox_to_3d_loc.py [hunk 3]
```diff
@@ -22,7 +22,27 @@
 
 
 class Node(AbstractNode):
-    """Node that used 2d bounding boxes information to estimate 3d location"""
+    """Node that uses 2d bounding boxes information to estimate 3d location.
```
[the remark this answers is not in the document]
Comment (leeping-ng) [reply peekingduck/pipeline/nodes/heuristic/bbox_to_3d_loc.py]: Updated

File: peekingduck/pipeline/nodes/heuristic/check_nearby_objs.py (hunk 1 above)
Comment (leeping-ng) [reply peekingduck/pipeline/nodes/heuristic/check_nearby_objs.py]: Updated

**Pull request merged**